data/yaml/venues.yaml

```diff
@@ -216,6 +222,10 @@ codeswitch:
 codi:
   acronym: CODI
   name: Workshop on Computational Approaches to Discourse
+codicrac:
+  acronym: CODICRAC
+  name: The CODI-CRAC 2022 Shared Task on Anaphora, Bridging, and Discourse Deixis
+    in Dialogue
 cogacll:
```

Comment:
This is a joint event, not a new venue. It should be handled by:

* Assigning it to the CODI venue ID (e.g., 2022.codi-1)
* Jointly listing it with CRAC (e.g., in joint.yaml, add `2022.codi-1` under `crac`)

Reply:
(Note this will require renaming the PDFs and any attachments)

Reply:
@mjpost I also added `2022.codi-crac` under `crac` in joint.yaml. Correct me if I'm wrong.